Include/internal/pycore_token.h

```diff
@@ -91,8 +91,7 @@ extern "C" {
                                  (x) == INDENT    || \
                                  (x) == DEDENT)
 #define ISSTRINGLIT(x)          ((x) == STRING           || \
-                                 (x) == FSTRING_MIDDLE   || \
-                                 (x) == FSTRING_END)
+                                 (x) == FSTRING_MIDDLE)
```

Comment:
@lysnikolaou I think this file is auto-generated so we might need to update the generate_tokens.py

Reply:
Oh, that's correct! Completely missed it. I've updated that as well.